bin/cask

```diff
@@ -371,6 +386,8 @@ def main():
                     sys.version_info))
         if len(sys.argv) > 1 and sys.argv[1] == 'exec':
             exec_command(sys.argv[2:])
+        elif len(sys.argv) > 1 and sys.argv[1] == 'emacs':
+            exec_emacs(sys.argv[2:])
```

Comment:
Fine for now, but please add a `# TODO:` comment that we should use a proper argument parser here at some point.